hist/hist/src/TGraph.cxx

```diff
@@ -1323,6 +1175,34 @@ TFitResultPtr TGraph::Fit(TF1 *f1, Option_t *option, Option_t *goption, Axis_t r
    return ROOT::Fit::FitObject(this, f1 , fitOption , minOption, goption, range);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Fit this graph with function with name fname.
+///
+/// interface to TGraph::Fit(TF1 *f1...
+///
+/// fname is the name of an already predefined function created by TF1 or TF2
+/// Predefined functions such as gaus, expo and poln are automatically
+/// created by ROOT.
+///
+/// fname can also be a formula, accepted by the linear fitter (linear parts divided
+/// by "++" sign), for example "x++sin(x)" for fitting "[0]*x+[1]*sin(x)"
```

Comment:
```suggestion
/// The `fname` parameter is the name of an already predefined function created by TF1 or TF2
/// Predefined functions such as gaus, expo and poln are automatically
/// created by ROOT.
///
/// The `fname` can also be a formula, accepted by the linear fitter (linear parts divided
/// by "++" sign), for example "x++sin(x)" for fitting "[0]*x+[1]*sin(x)"
```
I'd try to avoid starting sentences with lowercase